packages/datadog-ci/src/commands/trace/helper.ts

```diff
@@ -53,7 +53,9 @@ export abstract class CustomSpanCommand extends BaseCommand {
 
       return 1
     }
-    const ciSpanTags = getCISpanTags()
+    const realGithubJobName = getGithubJobNameFromLogs(this.context)
```

Comment:
this means that this is executed regardless of whether the user has set `DD_GITHUB_JOB_NAME`, right? This seems like a wasted effort, doesn't it? As the value is not used at all if the env var is there: https://github.com/DataDog/datadog-ci/pull/1911/files#diff-dc1fcf04816c2d334efaa97fa563c1431218db229a8e620d4023b5925304bafeR345